Command/GenerateProductCommand.php

```diff
@@ -37,24 +37,29 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $configFile = $input->getArgument('configuration-file');
+        $globalConfig = $this->getConfiguration($configFile, new ProductGeneratorConfiguration());
 
-        $config = $this->getConfiguration($configFile, new ProductGeneratorConfiguration());
+        $outputDir = $globalConfig['output_dir'];
+        $this->checkOutputDirExists($outputDir);
 
-        $generator = $this->getContainer()->get('pim_data_generator.generator.product');
+        $productGenerator = $this->getContainer()->get('pim_data_generator.generator.product');
 
-        $totalCount = $this->getTotalCount($config);
-
-        $outputDir = $config['output_dir'];
-
-        $output->writeln(
-            sprintf('<info>Generating <comment>%d</comment> products', $totalCount)
-        );
+        // TODO: is it wrong????
+        $totalCount = $this->getTotalCount($globalConfig);
 
+        $output->writeln(sprintf('<info>Generating <comment>%d</comment> products', $totalCount));
         $progress = $this->getHelperSet()->get('progress');
         $progress->start($output, $totalCount);
-
-        $generator->generate($config, $config['entities']['products'], $progress);
-
+        $productGenerator->generate($globalConfig, $globalConfig['entities']['products'], $progress);
         $progress->finish();
+
+        if (isset($globalConfig['entities']['product_drafts'])) {
+            $draftGenerator = $this->getContainer()->get('pim_data_generator.generator.product_draft');
+            $output->writeln(sprintf('<info>Generating <comment>%d</comment> product drafts', $totalCount));
+            $progress = $this->getHelperSet()->get('progress');
+            $progress->start($output, $totalCount);
+            $draftGenerator->generate($globalConfig, $globalConfig['entities']['product_drafts'], $progress);
+            $progress->finish();
+        }
```

Comment:
Maybe the products and the product_drafts generators should be moved into another class (like fixturesGenerator)